search/tests/mock_search_engine.py

```diff
@@ -321,20 +320,17 @@ def __init__(self, index=None):
         super(MockSearchEngine, self).__init__(index)
         MockSearchEngine.load_index(self.index_name)
 
-    def index(self, doc_type, sources):
+    def index(self, doc_type, sources):  # pylint: disable=arguments-differ
         """ Add/update documents of given type to the index """
-        if MockSearchEngine._disabled:
-            return None
```

Comment:
About style and best practice - therefore IMHO is no 'correct' way but 
I am personally in the favor or early return if requirements for executing it does not meet. I do not think this change was necessary.